content/_data/changelogs/weekly.yml

```diff
@@ -8209,6 +8209,286 @@
           title: security advisory
       authors:
         - wadeck
+    - type: bug
+      category: bug
+      pull: 4758
+      authors:
+      - Absh-Day
+      message: |-
+        Escape special characters from <code>href</code> attribute in console links.
+      references:
+        - issue: 62949
+        - url: https://www.jenkins.io/security/advisory/2020-07-15/#SECURITY-1945
+          title: related SECURITY-1945 advisory
```

Comment:
```suggestion
```

First time we would separately call out security fixes in the changelog that are listed in the advisory.

Reply:
Do you suggest to remove the public user bugfix and Jira issue from the changelog entirely? We have a case where we reference RFEs, so I do not think there is much difference for bugs:

![image](https://user-images.githubusercontent.com/3000480/88163024-5e317a80-cc12-11ea-89ba-f40727dfa4d6.png)